charts/kuberpult/setup-postgres.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env bash
+
+set -eu
+set -o pipefail
+
+function waitForDeployment() {
+  ns="$1"
+  label="$2"
+  echo "waitForDeployment: $ns/$label"
+  until kubectl wait --for=condition=ready pod -n "$ns" -l "$label" --timeout=30s;
+  do
+    sleep 4
+    echo "logs:"
+    kubectl -n "$ns" logs -l "$label" || echo "could not get logs for $label"
+    echo "describe pod:"
+    kubectl -n "$ns" describe pod -l "$label"
+    echo ...
+  done
+}
+
+earthly ../../infrastructure/docker/postgres+docker
+kind load docker-image postgres:local
```

Comment:
don't we need a `docker pull` here too?

Reply:
The image is generated from the above earthly command `earthly ../../infrastructure/docker/postgres+docker`